apps/omg_api/test/state/core_test.exs

```diff
@@ -542,14 +542,14 @@ defmodule OMG.API.State.CoreTest do
 
     state
     |> (&Core.exec(
-          Test.create_recovered([{@child_block_interval, 1, 0, alice}], eth(), [{alice, 7}]),
+          Test.create_recovered([{@child_block_interval, 0, 0, alice}], eth(), [{alice, 7}]),
```

Comment:
That was an error in the test. Those transactions (before change) did not spent the utxos  that were exited.